### PR TITLE
Dark mode auth

### PR DESCRIPTION
**Pull request by BelideNikhil**
## Features

+ Dark Mode with local storage
+ User Authentication

Preview: https://6257c9a028b6024520c1097b--off-road-tv.netlify.app/login

### CODE REVIEW

**Review by msdeshmukh009 — APPROVED** [review]
Very clean and readable code. The dark mode is 🔥. Good work!!

**Review by Varu98 — APPROVED** [review]
Nice work on the animation of the theme toggle btn 👍🏻

File: src/Components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+/*  */
+.nav-toggle-btn,
+.nav-search-mobile {
+    display: none;
+    display: none;
+}
+.header .theme-toggle-btn {
+    width: 28px;
+    height: 28px;
```
Comment (simran61) [inline src/Components/Navbar/Navbar.css:70]: try using relative units instead of px

**Review by simran61 — APPROVED** [review]
Nice UI. Just try to avoid using absolute units.
Clean folder structure and easily readable code

**Pull request merged**